core/src/main/java/in/testpress/fragments/EmptyViewFragment.kt

```diff
@@ -66,19 +66,20 @@ class EmptyViewFragment : Fragment() {
 
     fun displayError(exception: TestpressException) {
         when {
-            exception.isForbidden -> handleForbidden()
+            exception.isForbidden -> handleForbidden(exception)
             exception.isNetworkError -> handleNetworkError()
             exception.isPageNotFound -> handleIsPageNotFound()
             else -> handleUnknownError()
         }
     }
 
-    private fun handleForbidden() {
+    private fun handleForbidden(exception: TestpressException) {
+        val errorResponse: JSONObject? = JSONObject(exception.response.errorBody()?.string()!!)
         setEmptyText(
             R.string.permission_denied,
-                R.string.testpress_no_permission,
-                R.drawable.ic_error_outline_black_18dp)
-        retryButton.visibility = View.GONE
+            errorResponse!!.getString("detail"),
```

Comment:
Why error response is using double-bang operator? What if we don't pass any error body? This will crash the app.